doc/classes/Control.xml

```diff
@@ -14,6 +14,7 @@
 		Sets [member mouse_filter] to [constant MOUSE_FILTER_IGNORE] to tell a [Control] node to ignore mouse or touch events. You'll need it if you place an icon on top of a button.
 		[Theme] resources change the Control's appearance. If you change the [Theme] on a [Control] node, it affects all of its children. To override some of the theme's parameters, call one of the [code]add_theme_*_override[/code] methods, like [method add_theme_font_override]. You can override the theme with the inspector.
 		[b]Note:[/b] Theme items are [i]not[/i] [Object] properties. This means you can't access their values using [method Object.get] and [method Object.set]. Instead, use the [code]get_theme_*[/code] and [code]add_theme_*_override[/code] methods provided by this class.
+		[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/control_gallery.png]Screenshot of common Control-derived nodes with their names affixed[/url]
```

Comment:
Isn't it better to reference the page under `<link>` instead of an URL to the raw image?